### DISCUSSION TITLE
Request: Translations needed for a new string in interactive example macro

### DISCUSSION THREAD

**Discussion by schalkneethling** [mdn/mdn-community#79] [Localization]
Hey All!

Thanks to @SphinxKnight you can now localize the "Try It" heading added to interactive examples.
https://github.com/mdn/yari/pull/5874

Here is the relevant file to update:
https://github.com/mdn/yari/blob/main/kumascript/macros/EmbedInteractiveExample.ejs

When you have a moment, I am happy to review all pull requests. Thank you!

@mdn/yari-content-es , @mdn/yari-content-ja, @mdn/yari-content-ko , @mdn/yari-content-zh, @mdn/yari-content-ru, @mdn/yari-content-pt-br

**Comment by Graywolf9** [comment]
Thank you so much @schalkneethling, there is the PR for `es` locale

https://github.com/mdn/yari/pull/5879

**Reply by schalkneethling** [reply]
Thank you!

**Comment by myshov** [comment]
@schalkneethling thanks! Here is pr for `ru` locale

https://github.com/mdn/yari/pull/5891

**Comment by mfuji09** [comment]
@schalkneethling 
I have made PR for `ja` locale.

**Reply by schalkneethling** [reply]
Merged! Thank you.

**Comment by yin1999** [comment]
Hi @irvin. What do you think about `尝试一下` (translation of `try it`) for zh-CN.

**Comment by hochan222** [comment]
Thanks! Here is PR for `ko-locale`.

@mdn/yari-content-ko `try it`의 번역으로 `시도해보기` 로 작성했는데, 더 좋은 번역이 있을까요?

https://github.com/mdn/yari/pull/6192

**Reply by pje1740** [reply]
좋네요! 이견 없습니다 ㅎㅎ

**Reply by yechoi42** [reply]
좋습니다~

**Comment by dipikabh** [comment]
Closing this as it appears to be completed in all locales. Thanks, everyone!
